shared/src/main/scala/SugarMap.scala

```diff
@@ -0,0 +1,46 @@
+package vyxal
+object SugarMap:
+  def internalMap: Map[String, String] =
+    val out = Map.newBuilder[String, String]
+
+    val lowerUpdot = "abcdefghlmnoprsx" -> "ȧḃċḋėḟġḣŀṁṅȯṗṙṡṫẋ"
+    val upperUpdot = "ABCDEFGHLMNOPRSTWX" -> "ȦḂĊḊĖḞĠḢİĿṀṄȮṖṘṠṪẆẊ"
+    val upperDowndot = "BDHILMNORST" -> "ḄḌḤỊḶṂṆỌṚṢṬ"
```

Comment:
```suggestion
    val lowerUpdot = "abcdefghlmnoprstx" -> "ȧḃċḋėḟġḣŀṁṅȯṗṙṡṫẋ"
    val upperUpdot = "ABCDEFGHILMNOPRSTWX" -> "ȦḂĊḊĖḞĠḢİĿṀṄȮṖṘṠṪẆẊ"
    val upperDowndot = "BDHILMNORST" -> "ḄḌḤỊḶṂṆỌṚṢṬ"
```
well this doesn't seem right